File: 0485-max-consecutive-ones/0485-max-consecutive-ones.cs
public class Solution {
    public int FindMaxConsecutiveOnes(int[] nums) {
        int ret = 0;
        int curr = 0;
        
        foreach(int n in nums)
        {
            if(n == 0) {
                curr = 0;
                continue;
            }
            
            curr++;
            ret = Math.Max(ret, curr);
        }
        
        return ret;
    }
}

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.ts
function findMaxConsecutiveOnes(nums: number[]): number {
    let ret: number = 0;
    let curr: number = 0;
    
    for(const n of nums){
        if(n === 0){
            curr = 0;
            continue;
        }
        curr++;
        ret = Math.max(ret, curr);            
    }
    
    return ret;
};